packages/flutter_fuchsia/lib/example_main.dart

```diff
@@ -0,0 +1,21 @@
+import 'dart:io';
```

Comment:
Missing license header (see existing Dart files in this repo)

Reply:
Done.